python_modules/dagster/dagster/core/object_store.py

```diff
@@ -29,6 +31,19 @@ def ensure_boto_requirements():
 
 @six.add_metaclass(ABCMeta)
 class ObjectStore:
+    def __init__(self, types_to_register=None):
+        types_to_register = check.opt_dict_param(types_to_register, 'types_to_register')
```

Comment:
i think specifying `key_type` and `value_type` would be nice here from a documentation perspective

Reply:
👍